iRonCub-Mk3/calibrators/right_arm-calib.xml

```diff
@@ -19,13 +19,13 @@
 <group name="CALIBRATION">
 <param name="calibrationType">     10          10          10         10         5       12       12      7       7       6       6       6       6       6       6       6       </param>
 <!-- <param name="calibration1">        6576       -12866      -4585      -47529     1500    36903  -12703     0       0       0       0       0       0       0       0       0       </param> -->
-<param name="calibration1">        -4000        3000       3000      -4000    1500     63000    17600     0       0       0       0       0       0       0       0       0       </param>
-<param name="calibration2">	    0           0           0          0        16384     0      0        0       0      9102    9102    9102    9102    9102    9102    10000    </param>
+<param name="calibration1">       -4000        3000       3000      -4000    1500     63000    17600     0       0       0       0       0       0       0       0       0       </param>
+<param name="calibration2">	       0           0           0          0        16384     0      0        0       0      9102    9102    9102    9102    9102    9102    10000    </param>
 <param name="calibration3">        0           0           0          0          0       0       0        0       0       -1      1       -1      1       -1      1       1      </param>
 <param name="calibration4">        0           0           0          0          0       0       0       1475    1560    253     490     255     510     255     495     764     </param>
 <param name="calibration5">        0           0           0          0          0       0       0       2780    2306    19      120      0       172      0       62      190     </param>
-<param name="calibrationZero">     25       -8.4        -51.3       -5           0       0       0       0       0       0       0       0       0       0       0       0             </param>
-<param name="calibrationDelta">    0          0           0          0          0        0       0      15      0       0       0       0       0       0       0       0       </param>
+<param name="calibrationZero">     33          0       -51.3       -5           0       0       0       0       0       0       0       0       0       0       0       0             </param>
```

Comment:
I don't remember why we set `calibrationZero` to 33 deg here if everywhere else is 35 deg.

Reply:
might be a mistake, I'll double check

Reply:
this file is not used by the calibration since we use [right_arm-calib-iRonCub.xml](https://github.com/ami-iit/robots-configuration/pull/50/files#diff-15d9533e952527708d30306656d429afecf832c1a47e70a3b245aaee9ea3da88)
Therefore, regardless value you had written, we should never have observed any difference.

Reply:
> this file is not used by the calibration since we use [right_arm-calib-iRonCub.xml](https://github.com/ami-iit/robots-configuration/pull/50/files#diff-15d9533e952527708d30306656d429afecf832c1a47e70a3b245aaee9ea3da88)
> Therefore, regardless value you had written, we should never have observed any difference.

Thank you, I didn't notice it.